templates/samples.html

```diff
@@ -0,0 +1,229 @@
+{% extends "base.html" %}
+{% load verbatim %}
+
+{% block html_imports %}
+  <link rel="import" href="/static/elements/samples-imports{% if VULCANIZE %}.vulcanize{% endif %}.html">
+{% endblock %}
+
+{% block rss %}
+<link rel="alternate" type="application/rss+xml" href="http://www.chromestatus.com/features.xml" title="All features" />
```

Comment:
Feel free to disregard if this requires any work, but can we get an RSS feed of features that have samples "for free" by using parameters to `http://www.chromestatus.com/features.xml`? If so, can you use the appropriate URL/title for that RSS feed here?